StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1623UnitTests.cs

```diff
@@ -152,6 +152,39 @@ private class PrivateTestClass
             await this.VerifyCSharpFixAsync(testCode, fixedTestCode).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Verifies that empty summary tag does not throw an exception.
+        /// Regression test for #1943
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Fact]
+        public async Task EmptySummaryTagShouldNotThrowAnExceptionAsync()
+        {
+            var testCode = @"public class ClassName
+{
+    /// <summary></summary>
+    public int Property
+    {
+        get;
+    }
+}";
+
+            var fixedTestCode = @"public class ClassName
+{
+    /// <summary>Gets the value</summary>
```

Comment:
:exclamation: This suppresses the warning but does not actually fix the documentation problem. For other similar situations, we intentionally do not provide a code fix, and I think we should continue to follow that model here.